varia/sass/abstracts/_imports.scss

```diff
@@ -6,3 +6,4 @@
 @import "functions";
 @import "config-global";
 @import "mixins";
+@import "ie11-ponyfill";
```

Comment:
I don't like having this compiled to every child theme and loaded on every browser (even if it's harmless). I think I should move this to a separate stylesheet conditionally loaded for IE only and have that on the function that has the ponyfill too